lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -630,6 +630,8 @@ export class UserAgentApplication {
      *
      */
     acquireTokenSilent(userRequest: AuthenticationParameters): Promise<AuthResponse> {
+        this.logger.verbose("AcquireTokenSilent has been called");
+        this.logger.verbose(`AcquireTokenSilent params:\n${JSON.stringify(userRequest)}`);
```

Comment:
This could include Pii. I would argue it can be removed, since relevant parts of the request are logged downstream.

Reply:
Agreed. Removed.